website/docs/cloud-docs/api-docs/run-tasks/run-task-stages-and-results.mdx

```diff
@@ -20,13 +20,12 @@ page_title: Run Task Stages and Results - API Docs - Terraform Cloud and Terrafo
 
 Terraform Cloud uses run task stages and run task results to track [run task](/cloud-docs/workspaces/settings/run-tasks) execution.
 
-When Terraform Cloud creates a [run][], it reads the run tasks associated to the workspace. Each run task in the workspace is configured to begin during a specific [run stage](/cloud-docs/run/states). Terraform Cloud creates a run task stage object for each run stage that will trigger run tasks. Currently, you can only configure run tasks during the [Post-Plan Stage](/cloud-docs/run/states#3-the-post-plan-stage), so Terraform Cloud produces one `post-plan` run task stage for each run.
+When Terraform Cloud creates a [run][], it reads the run tasks associated to the workspace. Each run task in the workspace is configured to begin during a specific [run stage](/cloud-docs/run/states). Terraform Cloud creates a run task stage object for each run stage that will trigger run tasks. Currently, you can configure run tasks during the [Pre-Plan Stage](/cloud-docs/run/states#3-the-pre-plan-stage) and [Post-Plan Stage](/cloud-docs/run/states#5-the-post-plan-stage).
```

Comment:
We've been trying to stay away from words like "currently" that imply future functionality.

```suggestion
When Terraform Cloud creates a [run][], it reads the run tasks associated to the workspace. Each run task in the workspace is configured to begin during a specific [run stage](/cloud-docs/run/states). Terraform Cloud creates a run task stage object for each run stage that will trigger run tasks. You can configure run tasks during the [Pre-Plan Stage](/cloud-docs/run/states#3-the-pre-plan-stage) and [Post-Plan Stage](/cloud-docs/run/states#5-the-post-plan-stage).
```